setup.py

```diff
@@ -11,9 +19,97 @@ def extract_requirements(filename):
 
 install_requires = extract_requirements('requirements.txt')
 
+
+def read_module_contents():
+    with open('tendrl/node_agent/__init__.py') as app_init:
+        return app_init.read()
+
+
+def read_spec_contents():
+    with open('tendrl-node-agent.spec') as spec:
+        return spec.read()
+
+module_file = read_module_contents()
+metadata = dict(re.findall("__([a-z]+)__\s*=\s*'([^']+)'", module_file))
+version = metadata['version']
+
+
+class BumpCommand(Command):
+    """ Bump the __version__ number and commit all changes. """
+
+    user_options = [('version=', 'v', 'version number to use')]
+
+    def initialize_options(self):
+        new_version = metadata['version'].split('.')
+        new_version[-1] = str(int(new_version[-1]) + 1)  # Bump the final part
```

Comment:
Right, it bumps the minor version by default. You can use `--version` if you want to do a larger version bump.